packages/default-field-editors/package.json

```diff
@@ -35,7 +35,7 @@
     "@contentful/field-editor-radio": "^1.1.3",
     "@contentful/field-editor-rating": "^1.1.2",
     "@contentful/field-editor-reference": "^4.3.7",
-    "@contentful/field-editor-rich-text": "^1.1.16",
+    "@contentful/field-editor-rich-text": "^2.0.0-next.43",
```

Comment:
Is this considered a breaking change for `@contentful/default-field-editors` users?